templates/Layout/DNRoot_alerts.ss

```diff
@@ -13,18 +13,15 @@
 
 <div class="content">
 
-	<% if $AlertsConfigContent %>
-		<h3>Currently configured alerts</h3>
-		<pre>$AlertsConfigContent</pre>
+	<% if $AlertsConfigContent(master) %>
+		<h3>Currently configured alerts on master branch</h3>
+		<pre>$AlertsConfigContent(master)</pre>
 	<% end_if %>
 
 	<h3>Configuring alerts</h3>
 
 	<p>
-		Alerts are sent when a check against an
-		<a href="https://github.com/silverstripe-labs/silverstripe-environmentcheck/">environmentcheck module suite</a>
-		fails with a bad HTTP response status. You configure these checks by placing a <code>.alerts.yml</code> file in
-		the root of your site code.
+		Alerts are sent when a check HTTP Request against a URL fails with a non 200 HTTP response status. You configure these checks by placing a <code>.alerts.yml</code> file in the root of your site code.
```

Comment:
How about this simplified text? "Alerts are sent when a check fails with a non-200 HTTP response status. You configure these checks by placing a <code>.alerts.yml</code> file in the root of your site code."